packages/theme-data/src/baseTheme/components/flyout.js

```diff
@@ -0,0 +1,23 @@
+import { BORDER_RADIUS, COLOR, LENGTH } from "../../consts/types";
+
+export default {
+  "flyout.backgroundColor": {
+    type: COLOR,
+    value: { ref: "colorScheme.surfaceLevel100Color" }
+  },
+  "flyout.borderColor": {
+    type: COLOR
+  },
+  "flyout.borderRadius": {
+    type: BORDER_RADIUS,
+    value: { ref: "basics.borderRadii.large" }
+  },
+  "flyout.shadowBlur": {
+    type: LENGTH,
+    value: { ref: "basics.shadows.highBlur" }
+  },
+  "flyout.shadowColor": {
+    type: COLOR,
+    value: { ref: "colorScheme.highShadowColor" }
+  }
```

Comment:
Leave out `flyout.nubMargin` but let's include `flyout.nub.hypoteneuseLength` even if we don't use it in the react component. It's a valid part of the design language.